packages/gatsby-telemetry/src/event-storage.ts

```diff
@@ -103,11 +60,11 @@ export class EventStorage {
     }
   }
 
-  getConfig(key: string): string | boolean | Record<string, unknown> {
+  getConfig(key: string): string | boolean | unknown | Record<string, unknown> {
     if (key) {
       return this.config.get(key)
     }
-    return this.config.all
+    return this.config.all()
```

Comment:
🤔

Reply:
this was "wrong" before and after some changes typescript started to complain - https://github.com/gatsbyjs/gatsby/pull/39137/files/a3b9f5628f95c35e1b8f54cde9258708fc125ac7#diff-cca3d37894655cf103c925eb2e8d20ccdab86c9fc6e1093f60777acd06be4550L20 the config could be one of two types before, but in here I did remove ConfigStore (this would be stuff that write/read some settings to `~/.config/<some-tool>`) and kept just  MemoryConfig which interface wasn't actually compatible apparently